README.MD

```diff
@@ -9,286 +9,91 @@
 [![MIT license](https://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
 ![Release](https://img.shields.io/github/release/Microsoft/presidio.svg)
 
-## Description
+## What is Presidio
 
-Presidio *(Origin from Latin praesidium ‘protection, garrison’)* helps to ensure sensitive text is properly managed and governed. It provides fast ***analytics*** and ***anonymization*** for sensitive text such as credit card numbers, names, locations, social security numbers, bitcoin wallets, US phone numbers and financial data.
+Presidio _(Origin from Latin praesidium ‘protection, garrison’)_ helps to ensure sensitive text is properly managed and governed. It provides fast **_analytics_** and **_anonymization_** for sensitive text such as credit card numbers, names, locations, social security numbers, bitcoin wallets, US phone numbers and financial data.
 Presidio analyzes the text using predefined or custom recognizers to identify entities, patterns, formats, and checksums with relevant context. Presidio leverages docker and kubernetes for workloads at scale.
 
-### Why use presidio?
-
 Presidio can be integrated into any data pipeline for intelligent PII scrubbing. It is open-source, transparent and scalable. Additionally, PII anonymization use-cases often require a different set of PII entities to be detected, some of which are domain or business specific. Presidio allows you to **customize or add new PII recognizers** via API or code to best fit your anonymization needs.
 
 :warning: Presidio can help identify sensitive/PII data in un/structured text. However, because Presidio is using trained ML models, there is no guarantee that Presidio will find all sensitive information. Consequently, additional systems and protections should be employed.
 
-## Demo
-
-[Try Presidio with your own data](https://aka.ms/presidio-demo)
-
-## Features
-
-***Unstructured text anonymization***
+[Presidio Features](docs/overview.md#features)
 
-Presidio automatically detects Personal-Identifiable Information (PII) in unstructured text, annonymizes it based on one or more anonymization mechanisms, and returns a string with no personal identifiable data.
-For example:
+[Presidio Input and output](docs/overview.md#input-and-output)
 
-[![Image1](docs/assets/before-after.png)](docs/assets/before-after.png)
-
-For each PII entity, presidio returns a confidence score:
-
-[![Image2](docs/assets/findings.png)](docs/assets/findings.png)
-
-***Text anonymization in images*** (beta)
-
-Presidio uses OCR to detect text in images. It further allows the redaction of the text from the original image.
+## Demo
```

Comment:
Why not show the different images? To me graphics really help understand what a tool does.

Reply:
wanted to make the main readme minimal. I'll add this back, np

Reply:
So are we keeping the overview.md or not? I think at least one image is necessary on the readme. Many people prefer visual information over textual

Reply:
I kept the overview, added a link in the index and also added a new image in the main readme (I think too many images are a bit of overload...). Let me know how it looks now

Reply:
IMO the previous image was better. This one has too much details on it. WDYT?

Reply:
updated with new gif image